ext/spl/tests/gh11338.phpt

```diff
@@ -0,0 +1,47 @@
+--TEST--
+GH-11338 (SplFileInfo empty getBasename with more than on slash)
+--FILE--
+<?php
+
+function test($path) {
+    echo "Testing: $path\n";
```

Comment:
Could you add some quotes, otherwise the empty path case will have a trailing whitespace which IDEs will remove (or other tooling)
```suggestion
    echo "Testing: '$path'\n";
```